src/main/java/uk/gov/hmcts/ccd/service/MessagePublisherRunnable.java

```diff
@@ -59,14 +62,32 @@ private void processUnpublishedMessages(Pageable pageable,
         } else if (!processedEntities.isEmpty()) {
             messageQueueCandidateRepository.saveAll(processedEntities);
             log.info(String.format("%s Published %s messages to destination '%s'",
-                logPrefix, processedEntities.size(), publishMessageTask.getDestination()));
+                                   logPrefix, processedEntities.size(), publishMessageTask.getDestination()
+            ));
         }
     }
 
+    private String getPropertyValue(JsonNode data, MessageProperties property) {
```

Comment:
minor:
```suggestion
    private String getPropertyValue(JsonNode data, String propertyId) {
```
if it's not a big work to change this